nolearn/lasagne/base.py

```diff
@@ -164,6 +164,19 @@ def objective(layers,
               l1=0,
               l2=0,
               get_output_kw=None):
+    """
+    Default implementation of the NeuralNet Objective.
+    :param layers: The underlying layers of the NeuralNetwork
```

Comment:
Let's stick to calling it NeuralNet, it's what the class is called and more consistent.